LMS_Admin/src/main/java/com/SS/library/Utility/ConnectUtil.java

```diff
@@ -18,7 +18,7 @@
     public ConnectUtil() throws SQLException, ClassNotFoundException {
         Class.forName(driver);
         conn = DriverManager.getConnection(url,username,password);
-        conn.setAutoCommit(false);
+//        conn.setAutoCommit(false);

```

Comment:
Pramod's example has this as a protected method. What would be the pros and cons?